### DISCUSSION TITLE
不开个discord频道？

### DISCUSSION THREAD

**Discussion by SqrtTan** [MFn233/PCL-Mainpage#39] [一般]
感觉discord很有必要的样子（应该大部分人都有魔法吧

**Comment by MFn233** [comment]
我好像在kook频道里提到过一嘴discord版MFnArea

**Reply by SqrtTan** [reply]
发个链接？我没KOOK

**Reply by Ad-closeNN** [reply]
> 发个链接？我没KOOK

我记得他说过那个dc服务器太冷清了(?

**Comment by MFn233** [comment]
discord: https://discord.gg/yJBWuAgZ9m

**Comment by wuliaodexiaoluo** [comment]
《关于加进去隔天 Discord 频道列表就没有 MFn Area 这个服务器了这档子事》
